WebdevPeriod3/Models/ProductDto.cs

```diff
@@ -10,7 +10,7 @@ public class ProductDto
 	{
 		[Required(AllowEmptyStrings = false)]
 		public string Name { get; set; }
-		public List<string> SubSystems { get; set; }
+		public List<string> SubSystems { get; set; } = new List<string>();
```

Comment:
This is a pretty big commit. Remember to make small commits as it's easier to manage.